man/man8/zfs.8

```diff
@@ -268,6 +268,21 @@ zfs \- configures ZFS file systems
 .LP
 .nf
 \fBzfs\fR \fBdiff\fR [\fB-FHt\fR] \fIsnapshot\fR \fIsnapshot|filesystem\fR
+.fi
+
+.LP
+.nf
+\fB\fBzfs key -l\fR \fIfilesystem\fR | \fIvolume\fR\fR
```

Comment:
I think it would be easier to read (e.g. in a script or an email) if we also have long opts for the "verbs" here: 
```
zfs key --load
zfs key --unload
zfs key --change
```